docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md

```diff
@@ -170,3 +170,33 @@ Specify TLS versions via **Application config**, as shown in the following illus
 ![Application Configuration](/images/replicated/application-config.png)
 
 The recommended TLS version is 1.2.
+
+## Set up HTTP/HTTPS proxy
+
+You can set up HTTP and HTTPS proxies via **Application config**, and select **Enable Proxy** as per the following illustration:
+
+![Enable Proxy](/images/replicated/enable-proxy.png)
+
+When completing the **Enable Proxy** settings, keep in mind the following:
+
+- If your proxy is using the default ports for each protocol, then set the ports for the HTTP and HTTPS proxies to the default, 80 and 443 respectively, instead of 8080 and 8443 as shown in the preceding illustration.
+
+- If you have only one proxy set up (HTTP or HTTPS), then set the same values for both. This configuration sets operating system environment variables and Java system properties.
+
+- The help text for each field shows which Java system property or environment variable gets set by the field. System properties have the "-D" prefix. For example "Specify -Dhttps.proxyPort".
+
+- The **no proxy** fields are lists of exception hosts, provided as a comma-delimited list of addresses or hostnames. Include the following addresses:
+  - The Docker gateway address (172.17.0.1 by default).
+  - The address of any previously-specified web proxy.
+  - Any other IP addresses that you deem safe to bypass the proxy.
+
+- For the fields "HTTP no proxy setting" and "HTTP no proxy setting for Java", you need to add the docker gateway address (which is 172.17.0.1 by default).
```

Comment:
Doesn't this imply this must include this one Docker Gateway IP only. Maybe we can say something along the lines that this docker gateway IP must be present along with others from the last point. Actually, I think the previous bullet and this could be merged.

Reply:
Yes , good observation . Thanks @shristy